R/make_id_to_quant_scan_link_table.R

```diff
@@ -0,0 +1,216 @@
+#' Finding Links from PNNL's DMS
+#' 
+#' @description Link ID scans to quantification scans.
+#' * `make_id_to_quant_scan_link_table()`: Link ID scans to quant scans for 
+#'     multiple datasets.
+#' * `make_id_to_quant_scan_link_table_for_dataset()`: Link ID scans to quant
+#'     scans for a single dataset.
+#' @param scanStats placeholder
```

Comment:
same issue with placeholders here

Reply:
All parameters require documentation, so I just added "placeholder" because I'm not as familiar with these functions.